mr/src/test/java/org/apache/iceberg/mr/TestHelper.java

```diff
@@ -122,7 +144,7 @@ public DataFile writeFile(StructLike partition, List<Record> records) throws IOE
   }
 
   private GenericAppenderHelper appender() {
-    return new GenericAppenderHelper(table, fileFormat, tmp, conf);
+    return new GenericAppenderHelper(table, fileFormat, temp, conf);
```

Comment:
```suggestion
    return new GenericAppenderHelper(table, fileFormat, null != tmp ? tmp : temp, conf);
```

Reply:
because this needs to work for both Junit4+5 code

Reply:
oh this is clever, I should have thought of it. 
will update!

Reply:
unfortunately this throws errors like `Cannot resolve constructor 'GenericAppenderHelper(Table, FileFormat, Object, Configuration)`.
I'll just simplify it into an if statement.